tb_plugin/torch_tb_profiler/profiler/data.py

```diff
@@ -49,6 +52,7 @@ def __init__(self, worker, span=None):
         self.approximated_sm_efficiency_ranges = None  # Cached here. Will be processed to json on first trace view.
         self.blocks_per_sm_count = None
         self.occupancy_count = None
+        self.tid2tree = None
```

Comment:
how many time when we pickle/unpickle the whole tree?

Reply:
1 for pickle and 1 for unpickle